docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams add commits directly against the branch.
+
+   > **Note:** As an alternative, Dev teams can cut PRs against the branch, add What’s New content, collaborate with Reviewers, and merge the PR into the What's New branch.
+
+1. The technical writer periodically checks the What's New PR to determine how much content has been added.
+
+At this point, the larger Marketing org can review the What’s New content in the branch.
+
+### At the end of a release
+
+Complete the following steps in the days leading up to the release:
+
+1. Five days before the GA release, the PM or PMM who is running the release sends out a **Last Call** to dev teams to get their What’s New content in.
+
+1. Two days before the release, What’s New content is frozen.
+
+1. On release day, the technical writer merges the What’s New branch into main and backports the doc to the feature version branch.
+
+1. A PM reviews all content and adjusts wording, where appropriate.
+
+1. The technical writer provides a copy edit.
+
+1. The technical writers coordinates with Release Management and the GTM team for precise timing of when to merge the What's New doc into main.
+
+## How to determine if content belongs in a What's New document
+
+Grafana publishes a What’s New [docs page](//docs/grafana/latest/whatsnew/) and [blog post](https://grafana.com/blog/2022/11/29/grafana-9.3-release/) along with every minor and major release. These posts are popular, and a good way for users to learn about the exciting new things we’ve released. What’s New also drives our go-to-market enablement: we train the field and make videos on the topics in What’s New. However, unlike our comprehensive changelog, What’s New is curated. If it contained every update it would be too noisy for people to read, and if we wrote a detailed What’s New post for every little bug fix, we’d be wasting our time.
+
+So how do you decide whether to write a What’s New post for your latest improvement?
+
+### Add a What’s New for anything that could make a customer say “oo! 😃 ” or “ooo… 😬 ”
+
+What's new content should address changes that have some kind of material impact on the user experience.
+
+- Include changes that _affect customers_, whether they are new features to try out or important, long-requested bug fixes.
+  - Most visualization changes and most additions to the UI should be in the What's new document, even when they seem small. 
+- Almost every change or addition associated with Prometheus and Loki is of interest, too.
+- What's New content should also include changes that require customers to do something, like change their API keys to Service Accounts, or stop using a deprecated API or plugin.
+- A What's New doc should include announcements - things we want customers to notice and try out. These could also be notable community contributions where we want to thank a contributor.
```

Comment:
Not sure if this will fix the dash or not. It's rendering as an en, rather than em, dash.
```suggestion
- A What's New doc should include announcements — things we want customers to notice and try out. These could also be notable community contributions where we want to thank a contributor.
```